src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/components/HmrcUserResearchBanner.scala.html

```diff
@@ -38,8 +38,12 @@ <h2 class="govuk-heading-s hmrc-user-research-banner__title">
   </div>
   <button class="govuk-link hmrc-user-research-banner__close">
    @language match {
-    case Cy => {Dim diolch}
-    case _ => {No thanks}
+    case Cy => {<span aria-hidden="true">Dim diolch</span><span class="govuk-visually-hidden">
+      Dim diolch, nid wyf am gymryd rhan yn yr ymchwil defnyddwyr - cuddio'r neges hon
+    </span>}
+    case _ => {<span aria-hidden="true">No thanks</span><span class="govuk-visually-hidden">
+      No thanks, I do not want to take part in user research, hide this message
+    </span>}
```

Comment:
To avoid visual issues due to whitespace, these are on one line with only the visually hidden text on a new line